packages/circuit-ui/components/Popover/Popover.tsx

```diff
@@ -0,0 +1,267 @@
+/**
+ * Copyright 2021, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** @jsx jsx */
+import { css, jsx } from '@emotion/core';
+import { Theme } from '@sumup/design-tokens';
+import React, {
+  useState,
+  FC,
+  HTMLProps,
+  ReactNode,
+  SVGProps,
+  MouseEvent,
+  Fragment,
+  useMemo,
+  RefObject,
+} from 'react';
+import { useClickAway, useLatest } from 'react-use';
+import { Dispatch as TrackingProps } from '@sumup/collector';
+import { usePopper } from 'react-popper';
+import { Placement, State, Modifier } from '@popperjs/core';
+import { useTheme } from 'emotion-theming';
+
+import styled, { StyleProps } from '../../styles/styled';
+import { listItem, textMega } from '../../styles/style-mixins';
+import { useComponents } from '../ComponentsContext';
+import useClickHandler from '../../hooks/use-click-handler';
+import Hr from '../Hr';
+
+export interface BaseProps {
+  children: ReactNode;
```

Comment:
Should this not be a `string`? Should we accept other (custom) elements as PopoverItem labels?

Reply:
Usually, I like to keep props like these flexible. However, now that I think about it, I can't think of a use case when one would need to pass a custom element. You're right that it's better to restrict it to a string for now until a such a use case comes up.